### ISSUE TITLE
Rules and timezones make for a UX surprise

### ISSUE THREAD

**Issue by twobraids** [mozilla-iot/gateway#1428]
Times represented in Rules are always localized by the browser which leads to a UX bug.  Consider this scenario:

I have a Things Gateway running in US/Pacific timezone.   I travel to US/Eastern timezone and remember that I wanted to make a rule for my porch light.  I access my Things Gateway via the Internet and create a rule to turn the porch light on at 6pm.   On returning home, I notice my porch light going on at 3pm and looking at the rule confirms that it was written to say 3pm.

Rule times should always show as relative to the timezone of the Things Gateway location, not the timezone of where the user happens to be sitting. The user is given no warning that they must do the mental arithmetic to translate what the GUI is showing into the real time at home.  

I recognize that this might not be an easy fix - the Things Gateway has no concept of where it is, as it always uses UTC and depends on the presentation layer to localize times.

**Comment by hobinjk** [comment]
#1113 will help with this but a warning when rule time zone != local time zone might be good to have as well

**Comment by flatsiedatsie** [comment]
I just ran into this as well.

I created a rule to trigger the thermostat to go back to 19 degrees at 7:30.

This morning I woke up to a cold house, and noticed that the rule actually mentioned 9:30 now. This shift is exactly the 2 hour delta with UTC.

Same with the rule to turn the thermostat low at 00:01. It has turned into:

![rules-timeshift](https://user-images.githubusercontent.com/805405/58229640-c12f3500-7d21-11e9-92ac-88d47920f050.png)
